__docs__/wonder-blocks-typography/body-monospace.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import {Meta, StoryObj} from "@storybook/react";
+import packageConfig from "../../packages/wonder-blocks-typography/package.json";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import TypographyArgTypes from "./typography.argtypes";
+
+import {BodyMonospace} from "@khanacademy/wonder-blocks-typography";
+
+export default {
```

Comment:
**thought:** It would be great if we could describe what's the general purpose for each typography component, but this is not something we document anywhere, probably something worth sharing in the WB cubby meeting this week.

Reply:
Great idea! That would be helpful for usage - I'm not sure myself when certain typography components should be used (ex: when should the different label sizes be used?) I added it to our agenda!